### DISCUSSION TITLE
Is there a similar project to Medusa, but for movies?

### DISCUSSION THREAD

**Discussion by SHJordan** [pymedusa/Medusa#10115] [Q&A] [answered]
Looking for something to deal with all the intricacies of movies as Medusa does deal for tvshows and anime.

**Comment by p0psicles** [comment, marked as answer]
there is https://radarr.video

**Comment by SHJordan** [comment]
If anyone knows other alternatives let me know.

**Comment by nydave69** [comment]
> there is https://radarr.video

That might be an answer..

Wish there was a better one..   
Been running it on and off since it was started.. 
Have always had issues with it..
Design of the software seems to have an arbitrary flow to it..
